kotlin/src/main/kotlin/org/jetbrains/research/testspark/kotlin/KotlinPsiHelper.kt

```diff
@@ -85,9 +89,10 @@ class KotlinPsiHelper(private val psiFile: PsiFile) : PsiHelper {
         project: Project,
         classesToTest: MutableList<PsiClassWrapper>,
         caretOffset: Int,
-        maxPolymorphismDepth: Int, // check if cut has any non-java super class
+        maxPolymorphismDepth: Int, // check if cut has any non-java superclass
```

Comment:
I would change the word `non-java` to `user-defined`. I wondered for a few seconds what `non-java` means. Also, make this comment as a Javadoc before the variable:
```kotlin
/**
* How deep should the implementation check CUT's <i>user-defined superclasses</i> (i.e., non-standard Java classes).
*/
maxPolymorphismDepth: Int
```